packages/components/src/custom-select-control/test/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+/**
+ * WordPress dependencies
+ */
+import { CustomSelectControl } from '@wordpress/components';
+
+describe( 'CustomSelectControl', () => {
+	const onKeyDown = jest.fn();
+
+	it( 'Captures the keypress event and does not let it propagate', () => {
+		const wrapper = (
+			<div
+				role="none"
+				className="test-class-name"
+				onKeyDown={ onKeyDown }
+			>
+				<CustomSelectControl options={ [ 'a', 'b', 'c' ] } />
```

Comment:
The `options` prop should be an array of objects with a `key` and a `name` properties

```js
// ...
const options = [
  {
    key: 'one',
    name: 'Option one',
  },
  {
    key: 'two',
    name: 'Option two',
  },
  {
    key: 'three',
    name: 'Option three',
  },
];

render(
  <div role="none" onKeyDown={ onKeyDown }>
    <CustomSelectControl options={ options } />
  </div>
);
// ...
```